_data/social_media.yml

```diff
@@ -1,5 +1,5 @@
-- platform: "Github"
-  link: "https://github.com/18F" 
+- platform: "GitHub"
```

Comment:
This `platform` value is used in `social-media.html`: https://github.com/shuuji3/18f.gsa.gov/blob/abac7fa0ca2faac8caf5fffa8c2e2ab833ea46c8/_includes/social-media.html#L12-L12
and rendered as `18F on GitHub`.